presto-product-tests/conf/presto/etc/multinode-tls-worker.properties

```diff
@@ -14,7 +14,6 @@ discovery.uri=https://presto-master.docker.cluster:7778
 
 query.max-memory=1GB
 query.max-memory-per-node=512MB
-query.max-total-memory-per-node=1GB
```

Comment:
Similarly, for all the .properties files, I would recommend leaving the max-total-memory-per-node configuration.

Reply:
This change also follows `query.max-memory` and `query.max-total-memory` logic.
I think it's good to have them aligned. It would make sense to add `query.max-total-memory` and `query.max-total-memory-per-node` to test configs or remove them both.

Reply:
Makes sense.  We can leave it as is then.  thanks!